data/member_test.go

```diff
@@ -0,0 +1,78 @@
+package data
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/ubclaunchpad/rocket/model"
+)
+
+func TestMemberCreateGetUpdateRemove(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+	dal, cleanupFunc, err := newTestDBConnection()
+	assert.Nil(t, err)
+	defer cleanupFunc()
+
+	// Create a new member
+	member := &model.Member{
+		SlackID: "1234",
+		Name:    "Big Bruno",
```

Comment:
Minor: We should probably insert and update as many values for this member as possible, just to make sure that all our columns are configured correctly in the DB and Rocket. If you want you can leave this as-is for now and I can write more tests for this stuff later since you've already done the hard part of setting up the test DB in this PR.

Reply:
Yeah I wrote these tests just to make sure the test setup and rollback and everything was working fine, like you say all these tests could use some dedicated love in a separate PR